Clients/Xamarin.Interactive.Client.Mac/Main.storyboard

```diff
@@ -365,7 +374,7 @@ Gw
                                     <rect key="frame" x="0.0" y="0.0" width="450" height="200"/>
                                     <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                     <subviews>
-                                        <textView editable="NO" importsGraphics="NO" findStyle="panel" allowsCharacterPickerTouchBarItem="NO" allowsNonContiguousLayout="YES" textCompletion="NO" id="RJe-UZ-r3O">
+                                        <textView editable="NO" importsGraphics="NO" verticallyResizable="YES" findStyle="panel" allowsCharacterPickerTouchBarItem="NO" allowsNonContiguousLayout="YES" textCompletion="NO" id="RJe-UZ-r3O">
```

Comment:
We're going to have to decide how we want to handle this.   Just opening the storyboard inserts these changes.

Reply:
Assuming Xcode is just adding explicit defaults to objects in the graph, best practice would be to let it do so, but keep those automatic changes as a separate commit.

We have to just give in to it every so often so diffs stay usable/readable.

However, if you notice funky things like it changing frames or bounds, there's a good chance you did something unintentional in the UI and it might have moved everything by a pixel or something. Or deleted your constraints. If you see these types of changes it's best to run away.